README.md

```diff
@@ -1 +1,72 @@
-# Hello_world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello_world
+
+> Description the project.
+In this project, I set up a "Hello_world" repository.
+
+## Built With
+
+- Major languages
+  css, html 
+- Frameworks
+- Technologies used
+    VS code, git, github,
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Fundamental of HTML, CSS
+
```

Comment:
You are using the old readme template. :-1:  Kindly use the latest and new [readme template](https://github.com/microverseinc/readme-template) and make sure all header/sections are present. :1234:  You should also edit them to suit the project and your credentials :+1: